client/src/components/new-request/submit-request/reserve-submit.vue

```
@@ -46,12 +46,20 @@ export default class ReserveSubmitButton extends Vue {
     if (this.setup === 'cancel') {
       return 'Stop & Send To Examination'
     }
+
+    // delete the next 3 lines when re-enabling auto and conditional approvals
+    if (this.setup !== 'assumed') {
+      return 'Send to Examination'
+    }
+
     if (this.location !== 'BC' && this.setup !== 'assumed') {
       return 'Send for Examination'
     }
     switch (this.setup) {
+      // @ts-ignore - typescript knows setup can only === 'assumed' at this point and gives error
```

As mentioned above, there might be a solution to avoid suppressing the errors.